eng/mgmt/mgmtmetadata/sql_resource-manager.txt

```diff
@@ -3,12 +3,12 @@ AutoRest installed successfully.
 Commencing code generation
 Generating CSharp code
 Executing AutoRest command
-cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/sql/resource-manager/readme.md --csharp --version=latest --reflect-api-versions --csharp-sdks-folder=.
-2019-07-11 15:27:39 UTC
+cmd.exe /c autorest.cmd https://github.com/emgu-ms/azure-rest-api-specs/blob/failoverSwaggerChanges/specification/sql/resource-manager/readme.md --csharp --version=latest --reflect-api-versions --csharp-sdks-folder=E:\GitCode\azure-sdk-for-net\sdk
+2019-07-12 23:19:41 UTC
 Azure-rest-api-specs repository information
-GitHub fork: Azure
-Branch:      master
-Commit:      808cfe2da58aa6f1d93bed748004dfb97268e6e3
+GitHub fork: emgu-ms
```

Comment:
Aren't your changes checked into master? Published SDK must be based on azure/master.